src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball firstBall = new Ball();
+        Ball secondBall = new Ball();
+        Ball thirdBall = new Ball();
+
+        System.out.println("The first ball is " + firstBall.toString());
```

Comment:
Use for-loop instead

Reply:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-for-loop-for-creating-several-objects-of-the-same-class